ZConfig/components/logger/loghandler.py

```diff
@@ -25,6 +25,13 @@
 from logging.handlers import HTTPHandler, SMTPHandler
 from logging.handlers import NTEventLogHandler as Win32EventLogHandler
 
+# Export these, they're used in handlers.py
+SysLogHandler = SysLogHandler
+HTTPHandler = HTTPHandler
+SMTPHandler = SMTPHandler
+Win32EventLogHandler = Win32EventLogHandler
```

Comment:
- I find the reassignment useless, and would prefer a comment like `#noqa API` to make the intent clearer.
- I don't think we need to support backward-compatibility for re-exports of stdlib modules.